lua/kulala/init.lua

```diff
@@ -10,83 +10,87 @@ local ScriptsUtils = require("kulala.scripts.utils")
 local M = {}
 
 M.setup = function(config)
-  CONFIG.setup(config)
+	CONFIG.setup(config)
```

Comment:
Haven't realized I have an automatic formatting enabled. Will try to disable it.